addons/hi-res-thumbnails/addon.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Higher resolution thumbnails",
+  "description": "Replaces thumbnails across the website with higher resolution images. Useful on retina displays or browsing at >100% zoom.",
+  "info": [
+    {
+      "type": "warning",
+      "text": "Higher resolution images require larger filesizes. Take care not to accidentally incur data expenses if your network plan is limited.",
+      "id": "imagefilesize"
+    }
+  ],
+  "settings": [
+    {
+      "name": "Dimension Multiplier (%)",
+      "id": "multiplier",
+      "type": "integer",
+      "min": 100,
+      "max": 300,
+      "default": 200
+    },
+    {
+      "name": "Fix initially low-res thumbnails",
+      "id": "fixLowRes",
+      "type": "boolean",
+      "default": true
+    }
+  ],
+  "tags": ["community"],
```

Comment:
Maybe more tags would be better? I don't really know where a list of tags or when they're appropriate is, so I'd appreciate review from someone more experienced here :P

Reply:
A slightly outdated list (the "featured" tag is missing) is [here](https://scratchaddons.com/docs/reference/addon-manifest/#tags-array-required). This addon isn't for specific areas of the website and isn't a theme, so I don't think any other tags are necessary.

Reply:
Thanks for the link, and that makes sense to me!